accounts/backupGJAccount.php

```diff
@@ -61,8 +61,9 @@
 	}*/
 	//$query = $db->prepare("UPDATE `accounts` SET `saveData` = :saveData WHERE userName = :userName");
 	//$query->execute([':saveData' => $saveData, ':userName' => $userName]);
-	file_put_contents("../data/accounts/$accountID",$saveData);
-	file_put_contents("../data/accounts/keys/$accountID","");
+	$safeAccountID = basename($accountID);
```

Comment:
this doesn't hurt but it feels pointless when the value is guaranteed to be an int